### PR TITLE
Form group modifiers

### PR DESCRIPTION
**Pull request by tyom**
- Pass field modifier from EntitySearchForm
- Pass field modifier to `multiple-choice` field
- Render `initialOption` for MultipleChoiceField for `radio` type
- Add `light` and `smaller` modifier for form fields
- Add `actions` modifier for form-group container for submit button

image

### CODE REVIEW

File: src/templates/_macros/form.njk [hunk 1]
```diff
@@ -431,11 +438,28 @@
...
     {% set fieldChildren = props.children or [] %}
+    {% set modifier = props.modifier | concat('') | reverse | join(' c-multiple-choice--') if props.modifier %}
+
+    {% if props.type == 'radio' and props.initialOption %}
+      <div class="c-multiple-choice {{ modifier }}">
+        <input
+          class="c-multiple-choice__input"
+          type="{{ props.type }}"
+          name="{{ props.name }}"
+          id="{{ props.fieldId }}-0"
```
Comment (feedmypixel) [inline src/templates/_macros/form.njk]: potential for multiples of this on a page?

File: assets/stylesheets/components/form/_form-group.scss [hunk 2]
```diff
@@ -87,3 +87,24 @@
...
+
+  .c-form-control:focus,
+  .c-multiple-choice__input:checked + .c-multiple-choice__label::before,
+  .c-multiple-choice__input:focus + .c-multiple-choice__label::before {
+    border-color: $black;
+  }
+}
+
+.c-form-group--small {
+  font-size: 16px;
```
Comment (feedmypixel) [inline assets/stylesheets/components/form/_form-group.scss]: are we not using the `core-font` work?

File: assets/stylesheets/components/form/_form-group.scss [hunk 3]
```diff
@@ -87,3 +87,24 @@
...
+}
+
+.c-form-group--smaller {
+  font-size: 14px;
```
Comment (feedmypixel) [inline assets/stylesheets/components/form/_form-group.scss]: are we not using the `core-font` work?

File: assets/stylesheets/components/form/_form-group.scss (hunk 2 above)
Comment (tyom) [inline assets/stylesheets/components/form/_form-group.scss]: The plan is to update the font mixins to do less. They are doing to much, but they really should be just setting font size and line height for different screen sizes. I'm hardcoding them for now with the hope that we can address the typography globally very soon.

File: src/templates/_macros/form.njk (hunk 1 above)
Comment (tyom) [inline src/templates/_macros/form.njk]: This is always one as it is outside of the loop and is used for showing `initialOption` (see select box).

**Pull request merged**